src/main/java/pl/edu/agh/firecell/core/MenuScene.java

```diff
@@ -60,7 +58,13 @@ private SimulationConfig createInitialSimulationConfig() {
         Vector3i spaceSize = new Vector3i(3, 3, 3);
         List<Cell> cells = IntStream.range(0, 9)
                 .mapToObj(flatIndex -> IndexUtils.expandIndex(flatIndex, spaceSize))
-                .map(cellIndex -> new Cell(random.nextDouble(), random.nextDouble()))
+                .map(cellIndex -> new Cell(
```

Comment:
I think it's unnecessary to split these arguments into multiple lines - they will fit in one line without a problem.